api_examples/graphql/queries/gcp_project_import.graphql

```diff
@@ -0,0 +1,116 @@
+# Description
+# -----------
+#
+# The mutation query import GCP project to a Turbot Folder. The mutation query needs to be run in 2 steps.
+# STEP 1 is to create a GCP project inside the Turbot Folder.
+# STEP 2 is to use the Turbot ID that is returned from STEP 1 as resource ID for setting client email and privet key.
+
+# Prerequisites
+# -------------
+#
+# Create a service account in IAM for the project that needs to be imported. 
+# After the service account has been created, assign the following roles: owner,viewer,Pub/Sub Admin and Logging Admin.
```

Comment:
owner,viewer,Pub/Sub Admin and Logging Admin. <- commas too close

Mix of upper and lower case, please sort this out.